doc/api/stream.md

```diff
@@ -432,6 +432,19 @@ file.end('world!');
 // Writing more now is not allowed!
 ```
 
+[`stream.end()`][stream-end] will call it's callback and emit `'error'`
+with (in order of precedence):
+* `ERR_STREAM_WRITE_AFTER_END` if `chunk` is not nully and
+[`stream.end()`][stream-end] has been called.
```

Comment:
Nit: perhaps add a little indent (here and below in this list)?
```suggestion
  [`stream.end()`][stream-end] has been called.
```

Reply:
I'm confused. Why? Is that a markdown thing?

Reply:
I'm +1 on @lundibundi, it's easy to read and recognize where is the items in this list :)

Reply:
Fixed